finetuner-workflow/finetuner/Dockerfile

```diff
@@ -12,5 +12,6 @@ RUN pip3 install --no-cache-dir -r requirements.txt
 COPY ds_config.json .
 COPY finetuner.py .
 COPY evaluator.py .
+COPY inference.py .
```

Comment:
I'm fine with doing this to get everything working ASAP, but we should be making the inference and finetuner images separate.

We at least don't need `deepspeed` in the inference image, and there's probably some other unused dependencies as well.

This is also a general huggingface-transformers-kserve inference script. Should this live somewhere separate like `ml-images`? I think we've been in need of putting all our the nested images in kubernetes-cloud someone else more visible.